wolfssl/openssl/ssl.h

```diff
@@ -1189,6 +1189,9 @@ typedef WOLFSSL_SRTP_PROTECTION_PROFILE      SRTP_PROTECTION_PROFILE;
 #define SSL_CTX_get_app_data(ctx)       wolfSSL_CTX_get_ex_data(ctx,0)
 #define SSL_CTX_set_app_data(ctx,arg)   wolfSSL_CTX_set_ex_data(ctx,0, \
                                                                   (char *)(arg))
+#define SSL_CTX_set_dh_auto(ctx, arg) \
+        wolfSSL_CTX_ctrl(ctx,SSL_CTRL_SET_DH_AUTO,arg,NULL)
```

Comment:
Since wolfSSL_CTX_ctrl doesn't have a case statement entry for SSL_CTRL_SET_DH_AUTO, it will hit the default case which will print an error and return WOLFSSL_FAILURE.  Even if this is a no-op, a case for it should be added to wolfSSL_CTX_ctrl so we return success.